src/delphi/constants.py

```diff
@@ -1,7 +1,9 @@
 from importlib.resources import files
+from pathlib import Path
+from typing import cast
 
 STATIC_ASSETS_DIR = files("delphi.static")
-CONFIG_PRESETS_DIR = STATIC_ASSETS_DIR / "configs"
+CONFIG_PRESETS_DIR = cast(Path, STATIC_ASSETS_DIR / "configs")
```

Comment:
Is this a Path?

Reply:
It is now. More generally, it is a Path in practice, and doing this casting here (1) breaks nothing and (2) makes some basic operations more straightforward with less boilerplate.